src/config/steps/fing.coffee

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  name: 'fing',
-  view : 'steps/fing'
-  isActive: (instance) ->
-    instance.attributes \
-      && instance.attributes.context \
-      && instance.attributes.context.toLowerCase() is 'fing'
+    name: 'fing',
+    view : 'steps/fing'
+    isActive: (instance) ->
+        instance.attributes \
```

Comment:
@lespacedunmatin Sorry I missed your comment. The thing is that the whole app is indented this way and we need to keep consistency.